src/runner.ts

```diff
@@ -218,23 +216,22 @@ export class Runner {
         for (const file of files) {
             if (!fs.existsSync(file)) {
                 console.error(`Unable to open file: ${file}`);
-                onComplete(1);
-                return;
+                return onComplete(1);
             }
 
             const buffer = new Buffer(256);
-            buffer.fill(0);
             const fd = fs.openSync(file, "r");
             try {
                 fs.readSync(fd, buffer, 0, 256, 0);
-                if (buffer.readInt8(0) === 0x47 && buffer.readInt8(188) === 0x47) {
+                if (buffer.readInt8(0, true) === 0x47 && buffer.readInt8(188, true) === 0x47) {
                     // MPEG transport streams use the '.ts' file extension. They use 0x47 as the frame
                     // separator, repeating every 188 bytes. It is unlikely to find that pattern in
                     // TypeScript source, so tslint ignores files with the specific pattern.
                     console.warn(`${file}: ignoring MPEG transport stream`);
-                    return;
+                    fs.closeSync(fd);
+                    continue;
                 }
-            } finally {
+            } catch (e) {
```

Comment:
good catch, I thought finally would not be executed when leaving the try block with `return` or `continue`. Will revert that change